src/components/elements/Map-mapbox/MapControls/ImageControl.tsx

```diff
@@ -0,0 +1,99 @@
+import { t } from "@transifex/native";
+import classNames from "classnames";
+import { Dispatch, SetStateAction } from "react";
+
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
+import ModalAdd from "@/components/extensive/Modal/ModalAdd";
+import { uploadImageData } from "@/components/extensive/Modal/ModalContent/MockedData";
+import { useModalContext } from "@/context/modal.provider";
+
+import { VARIANT_FILE_INPUT_MODAL_ADD_IMAGES } from "../../Inputs/FileInput/FileInputVariants";
+import Text from "../../Text/Text";
+
+const ImageControl = ({
+  viewImages,
+  setViewImages
+}: {
+  viewImages: boolean;
+  setViewImages: Dispatch<SetStateAction<boolean>>;
+}) => {
+  const { openModal, closeModal } = useModalContext();
+
+  const openFormModalHandlerUploadImages = () => {
+    openModal(
+      <ModalAdd
+        title="Upload Images"
+        variantFileInput={VARIANT_FILE_INPUT_MODAL_ADD_IMAGES}
+        descriptionInput="Drag and drop a geotagged or non-geotagged PNG, GIF or JPEG for your site Tannous/Brayton Road.."
+        descriptionList={
+          <Text variant="text-12-bold" className="mt-9 ">
+            Uploaded Files
```

Comment:
Wrap free text in this component in `t()` please.

Reply:
done